static/app/views/dashboards/widgetBuilder/components/visualize.tsx

```diff
@@ -693,7 +710,7 @@ function AggregateParameter({
     const inputProps = {
       required: parameter.required,
       value:
-        parameter.value ?? ('defaultValue' in parameter && parameter?.defaultValue) ?? '',
+        currentValue ?? ('defaultValue' in parameter && parameter?.defaultValue) ?? '',
```

Comment:
Also snuck in this change. I realized that when I would update the value in `count_if(transaction.duration,equals,300)`, the text field wouldn't update and this is why